google_nest_sdm/webrtc_util.py

```diff
@@ -0,0 +1,114 @@
+# SDP Direction Constants
+DIRECTION_SENDRECV = 'sendrecv'
+DIRECTION_SENDONLY = 'sendonly'
+DIRECTION_RECVONLY = 'recvonly'
+DIRECTION_INACTIVE = 'inactive'
+
+AVAILABLE_DIRECTIONS = [DIRECTION_SENDRECV,
+                        DIRECTION_SENDONLY,
+                        DIRECTION_RECVONLY,
+                        DIRECTION_INACTIVE]
+
+
+class WebRTCUtility:
```

Comment:
Remove the class and just make the functions module methods e.g. `def get_media_direction(sdp, kind):`

... unless it would be useful to add some state in the class (e.g. don't run the same string parsing multiple times like splitting)